third_party/dummy-hook/CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
-if(EXISTS "${Ginkgo_SOURCE_DIR}/.git")
+execute_process(COMMAND ${GIT_EXECUTABLE} rev-parse --absolute-git-dir
+        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+        OUTPUT_VARIABLE Ginkgo_GIT_DIR
```

Comment:
in the first cmake from scratch, the Ginkgo_GIT_DIR is empty

Reply:
Sorry about the confusion, but this PR is still WIP. It needs https://github.com/ginkgo-project/git-cmake-format/pull/6 and some additional changes to actually work

Reply:
I change the commit and the repo in local for testing.
you need to add `find_package(Git QUIET)`
the first run will not include Git yet, which is included in ginkgo_git_information, so the GIT_EXECUTABLE is notfound
Thus, git is available in the second time.
and maybe you also need to change the ginkgo_git_information there.